src/app/App.tsx

```diff
@@ -28,12 +29,20 @@ const switchClassname = style({
 });
 
 const App: React.ComponentType = () => {
+
     const intl = useIntl();
     const { rbac, server, isOrgAdmin } = useApp();
     const location = useLocation();
     const insights = getInsights();
     const [ usingExperimental, setUsingExperimental ] = React.useState<boolean>(false);
 
+    const url = window.location.pathname;
+    const filename = url.substring(url.lastIndexOf('/') + 1);
+    const userPreferences = <a href={ localUrl(`/user-preferences/notifications/${ filename }`,
+        getInsights().chrome.isBeta()) }> User Preferences </a>;
+    const myUserAccess = <a href={ localUrl(`/settings/my-user-access?bundle=${ filename }`,
```

Comment:
I was checking, and I think we can fix this `bundle=` to always be `rhel`... Notifications permissions are part of the RHEL bundle.